token_classification.py

```diff
@@ -143,7 +143,7 @@ def compute_metrics(evaluation_preds):
 # Performing inference
 text = "The Golden State Warriors are an American professional basketball team based in San Francisco."
 # We need to tokenize the inputs and turn them to PyTorch tensors
-encoded_input = tokenizer(text, return_tensors="pt")
+encoded_input = tokenizer(text, return_tensors="pt").to("cuda")
```

Comment:
Let's do `.to(trainerargs..device)` since the `Trainer` is device agnostic via accelerate